deployment/build_and_release/live/prod/terragrunt.hcl

```diff
@@ -12,7 +12,9 @@ inputs = merge(
   {
     bucket_count = 2
 
-    cloudbuild_trigger_tag = ".*"
+    # Require "v" in front of the regex from
+    # https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
+    cloudbuild_trigger_tag = "^v(?P<major>0|[1-9]\\d*)\\.(?P<minor>0|[1-9]\\d*)\\.(?P<patch>0|[1-9]\\d*)(?:-(?P<prerelease>(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+(?P<buildmetadata>[0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"
```

Comment:
Yikes! This seems more advanced than we'd need as it has capture groups and supports all of the stuff after the patch version too. OTOH, this is a "standard" regexp and perhaps deviating is more risky. @AlCutter thoughts?

Reply:
I switched over to the shorter one (and realized that that's actually the one I tested with).